docs/fos_http_cache.md

```diff
@@ -1,21 +1,30 @@
 # FOSHttpCacheBundle usage in eZ
 
-Introduced with eZ Publish 5.4, [FOSHttpCacheBundle][fos] is supported by eZ Platform, and covers the following
-features:
+_This doc explains how this bundle integrates with FOSHttpCacheBundle internally, and is aimed at core developers._
 
-* Tags purging/baning (since version 1.8)
-* Http cache purge/ban
-* User context hash
+
+eZ Platform HttpCache _(this bundle)_ uses [FOSHttpCacheBundle 1.x][fos] in the following way:
+- When `local` purge type is set, system uses a custom Symfony Proxy tag aware data store, and hence does not
+  use FOSHttpCacheBundle directly. However FOSHttpCache(Bundle) can still be used to manipulate responses, incl tagging.
+- When `http` purge type is set, system uses FOSHttpCache configured with varnish as proxy client.
+
+As HttpCache in eZ Publish/Platform pre dates FOSHttpCache, we have own abstractions for:
+ - purge clients: In this bundle exposed as `ezplatform.http_cache.purge_client` service
+ - App Cache: `EzSystems\PlatformHttpCacheBundle\AppCache` app cache class to extend from, this in turn extends
+   `FOS\HttpCacheBundle\SymfonyCache\EventDispatchingHttpCache` however provides own set of request aware purgers and
+   own handling of user context hash which for BC reasons is called `X-User-Hash` in eZ.
+
+_Note: Once FOSHttpCache(Bundle) has full support for tagging we might se a major version of this bundle be refactored to more
```

Comment:
`Once FOSHttpCache(Bundle) has full support for tagging, a major version of this bundle might be be refactored to more`